.travis.yml

```diff
@@ -1,8 +1,33 @@
 osx_image: xcode10
 language: objective-c
-podfile: Demo/Podfile
-cache: cocoapods
+xcode_project: RxNimble.xcodeproj
+xcode_scheme:
+  - RxNimbleRxBlocking
+  - RxNimbleRxTest
+
+cache:
+  directories:
+    - $HOME/Library/Caches/org.carthage.CarthageKit/dependencies
+    - Carthage
+
 before_install:
-  - gem install cocoapods -v '1.3.1'
-  - pod repo update
-script: cd Demo ; set -o pipefail && xcodebuild -workspace 'Demo.xcworkspace' -scheme 'Demo' -configuration 'Debug' -sdk iphonesimulator -destination platform='iOS Simulator',OS='12.0',name='iPhone 8' build test  | xcpretty -c --test
+  - carthageversion=$(carthage version)
+  - if [ $carthageversion != "0.31.2" ]; then brew uninstall carthage; HOMEBREW_NO_AUTO_UPDATE=1 brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/a85feeb75bc9e9beb7f2e9dc6e2ccc996a6aeaf5/Formula/carthage.rb; fi
```

Comment:
to my knowledge, a carthage version can't be specified in a Gemfile, so this is currently the fastest way I know to install a specific version. And actually I didn't mean to use 0.31.2 specifically, I'd like to always use the most up-to-date one but `brew update` is expensive

Reply:
Yeah this is a bit backwards but I don't have a nicer idea to go around this for now :)